src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.iOS.cs

```diff
@@ -1251,7 +1318,7 @@ private void PropagateUnusedSpaceRelayout()
 
 		internal void NeedsRelayout()
 		{
-			var nativePanel = CollectionView as NativeListViewBase;
+			var nativePanel = (CollectionView as NativeListViewBase)!;
```

Comment:
Could this fail another fail, e.g. if CollectionView is a getting null because of some memory constraint?

Reply:
Yes indeed. That's a possible scenario. I will make the adjustments

Reply:
Then I guess we should account for that here too: https://github.com/unoplatform/uno/blob/a505f6fcc3cd02a7723d2ab2152f9137fb4e8b76/src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.iOS.cs#L71

And btw, the nativePanel is basically the `Owner` defined in the line above right?

@jeromelaban

Reply:
yes that should be the same

Reply:
I did the change. Thanks!